src/democracyworks/smartystreets/core.clj

```diff
@@ -0,0 +1,25 @@
+(ns democracyworks.smartystreets.core
+  (:require
+   [clj-http.client :as http]))
+
+(defrecord Client [auth-id auth-token])
+
+(defn client
+  "Returns a SmartyStreets client."
+  [auth-id auth-token]
+  (->Client auth-id auth-token))
+
+(defn http-get
```

Comment:
perhaps `get` and `post` should be protocol methods on the client? it doesn't make a difference either way, but it might be nice to clarify the api.

Reply:
I actually had `fetch-one` and `fetch-many` in the API namespaces defined as a protocol and extended `Client` to each of them, to clarify the API. Switching back to that would not be a breaking change, if that's desired.

Here, `http-get` and `http-post` are little more than convenience wrappers with some default options, so it's implementation detail (specific to Client) and not API.